src/Jmikola/AutoLogin/Http/Firewall/AutoLoginListener.php

```diff
@@ -34,11 +35,11 @@ class AutoLoginListener implements ListenerInterface
      * @param AuthenticationManagerInterface                 $authenticationManager
      * @param string                                         $providerKey
      * @param string                                         $tokenParam
-     * @param LoggerInterface                                $logger
+     * @param PsrLoggerInterface|HttpKernelLoggerInterface   $logger
```

Comment:
I suggest dropping support for Symfony 2.1 and 2.2 (which are long dead anyway), which will allow to always typehint the PSR-3 logger (the Symfony interface extends the PSR one since 2.3)